consumer_test.go

```diff
@@ -315,6 +340,94 @@ var _ = Describe("Consumer", func() {
 		Expect(uniques).To(HaveLen(15000))
 	})
 
+	It("should consume/commit/reset/resume", func() {
```

Comment:
so I am not really sure we need another fuzzing test for this. I don't see the added value. what you have done in https://github.com/bsm/sarama-cluster/pull/197/files#diff-e85d03976d9766c4d00ac176511d4f06R188 seems enough, please remove that test.

Reply:
Removed the test. This was meant to check that we do actually rewind and the first offset is actually the one we expect. The earlier fuzzing test actually doesnt deal specifically with start offsets, right? 

Rather it deals with consuming atleast fixed number of unique entries (partition/offset pairs)?

I have removed it as you advised. But, let me know if we need check the start offsets when we start consumption.